### PR TITLE
Add stack/vstack/hstack converter

### PR DESCRIPTION
**Pull request by disktnk**
- reorder mapping keys to be same order with `functions/__init__.py`
- add stack, use `Unsqueeze` + `Concat`
- add vstack, hstack

### CODE REVIEW

File: onnx_chainer/functions/array.py [hunk 1]
```diff
@@ -407,3 +407,34 @@ def convert_ResizeImages(func, opset_version, input_names, num_outputs,
...
+
+
+def convert_Stack(func, opset_version, input_names, num_outputs, context,
+                  parameters):
+    gb = onnx_helper.GraphBuilder()
+    axis = func.axis
+    if axis < 0:
+        axis = len(func.inputs[0].shape) + 1 + axis
+
+    # To use concat op, reshape every inputs add new axis
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: s/add new axis/to add new axes/? (singular/plural mismatch)

File: onnx_chainer/functions/array.py [hunk 2]
```diff
@@ -407,3 +407,34 @@ def convert_ResizeImages(func, opset_version, input_names, num_outputs,
...
+    inputs = [gb.op('Unsqueeze', [name], axes=[axis]) for name in input_names]
+    gb.op('Concat', inputs, axis=axis)
+    return gb.nodes()
+
+
+def convert_Hstack(func, opset_version, input_names, num_outputs, context,
+                   parameters):
+    gb = onnx_helper.GraphBuilder()
+    axis = 0 if len(func.inputs[0].shape) == 1 else 1
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: Even scalars can be hstacked? We need Unsqueeze for this case (I doubt this feature is useful, though...)?
```
>>> F.hstack([np.array(3), np.array(4)])
variable([3, 4])
```

File: onnx_chainer/functions/array.py [hunk 3]
```diff
@@ -407,3 +407,34 @@ def convert_ResizeImages(func, opset_version, input_names, num_outputs,
...
+    gb.op('Concat', input_names, axis=axis)
+    return gb.nodes()
+
+
+def convert_Vstack(func, opset_version, input_names, num_outputs, context,
+                   parameters):
+    gb = onnx_helper.GraphBuilder()
+    inputs = input_names
+    if len(func.inputs[0].shape) == 1:
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: Need one more Unsqueeze for scalars?

File: onnx_chainer/functions/array.py (hunk 2 above)
Comment (disktnk) [reply onnx_chainer/functions/array.py]: add test cases about input array is scalar, and support them both hstack and vstack

**Comment by disktnk** [comment]
ChainerCI, test this please

**Pull request merged**